docs/changelog.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+## v1.0.0
```

Comment:
Can we just omit `v` for Change Log entries (so use `1.0.0`) and keep the `v` prefix for tags?

Reply:
ok